scala-native-jdbc/src/main/scala/com/github/lolgab/jdbc/SimpleResultSet.scala

```diff
@@ -0,0 +1,33 @@
+package com.github.lolgab.jdbc
+
+import java.sql.ResultSet
+
+/** Helper trait for result sets, with some default implementations */
+trait SimpleResultSet extends ResultSet {
```

Comment:
Added this helper to add some default implementations, which can make creating new drivers a bit easier.

Those can always be overriden anyway.